Link error on Windows with Rust 1.33.0

Thanks for this useful crate.

After upgrading Rust to 1.33.0 I get the following link error from `nfd` when building [Cinter](https://github.com/askeksa/Cinter):
```
libnfd-ec2bb1b08ec0519c.rlib(nfd_win.o) : error LNK2019: unresolved external symbol __imp_SHCreateItemFromParsingName referenced in function "enum nfdresult_t __cdecl SetDefaultPath(struct IFileDialog *,char const *)" (?SetDefaultPath@@YA?AW4nfdresult_t@@PEAUIFileDialog@@PEBD@Z)
```
1.32.0 works fine.

I'm on Windows 10 with Visual Studio 2017.

--- a/build.rs
+++ b/build.rs
@@ -46,6 +46,7 @@ fn main() {
         cfg.file(nfd!("nfd_win.cpp"));
         cfg.compile("libnfd.a");
         println!("cargo:rustc-link-lib=ole32");
+        println!("cargo:rustc-link-lib=shell32");
         // MinGW doesn't link it by default
         println!("cargo:rustc-link-lib=uuid");
     } else {

